docs/LEARNING.md

```diff
@@ -1,3 +1,3 @@
 # Learning
 
-Ballerina [learn](https://ballerina.io/learn) page contains multiple resources you to get up to speed with the language, master the concepts, and build comprehensive real-world use cases. 
\ No newline at end of file
+The Ballerina [learn](https://ballerina.io/learn) section contains multiple resources for you to get up to speed with the language, master the concepts, and build comprehensive real-world use cases. 
```

Comment:
Page is better than section.

Reply:
Also add a list of links for key resources.

Reply:
We have the resources listed in the `RESOURCES.md `file. Will add a link to it.

Reply:
Done.